beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactory.java

```diff
@@ -120,6 +127,17 @@ public BlockOperationSelectorFactory(
     this.graffitiBuilder = graffitiBuilder;
     this.forkChoiceNotifier = forkChoiceNotifier;
     this.executionLayerBlockProductionManager = executionLayerBlockProductionManager;
+    this.dataColumnSidecarComputationTimeSeconds =
+        new MetricsHistogram(
+            metricsSystem,
+            timeProvider,
+            TekuMetricCategory.BEACON,
+            "data_column_sidecar_computation_seconds",
+            "Time taken to compute data column sidecar, including cells and inclusion proof (histogram)",
```

Comment:
Description looks outdated, we don't calculate cell proofs anymore, we get it from the EL and it's a part of blob tx

Reply:
My bad t's only inclusion proof. all good